spec/support/acceptance/purge.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+def purge_node
+  case fact('osfamily')
+  when 'Debian'
+    on default, 'apt-get purge -y libnode*', { acceptable_exit_codes: [0, 100] }
```

Comment:
when you purge things, `libnodeXX` remains on the system and conflicts with files from nodesource packages :roll_eyes: